docs/aquacomputer_d5next.rst

```diff
@@ -109,6 +109,7 @@ Sysfs entries
 =========================== ==============================================================
 temp[1-20]_input            Physical/virtual temperature sensors (in millidegrees Celsius)
 temp[1-4]_offset            Temperature sensor correction offset (in millidegrees Celsius)
+temp[1-4]_max               Temperature sensor alarm setpoint (in millidegrees Celsius)
```

Comment:
Should be `temp1_max`, only one such entry is generated.

Reply:
nice catch, pushed